charts/spire/charts/spire-server/templates/tornjak-config.tpl

```diff
@@ -0,0 +1,38 @@
+{{- if eq (.Values.tornjak.enabled | toString) "true" }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tornjak-config
+  namespace: {{ .Values.namespace }}
+data:
+  server.conf: |
+    server {
+      metadata = "insert metadata"
+    }
+
+    plugins {
+    {{- if .Values.tornjak.config }}
+    {{- if .Values.tornjak.config.backend }}
+    {{- if .Values.tornjak.config.backend.dataStore }}
+      DataStore "sql" {
+        plugin_data {
+          drivername = "{{ .Values.tornjak.config.backend.dataStore.driver }}"
+          # TODO is this a good location?
+          filename = "{{ .Values.tornjak.config.backend.dataStore.file }}"
+        }
+      }
+      {{- end }}
+
+      {{- if .Values.tornjak.config.enableUserMgmt }}
+      UserManagement "KeycloakAuth" {
```

Comment:
Nice. :)

Reply:
Are there non KeycloakAuth UserManagement plugins? If so, should the config take that into account?

Reply:
This should work with any Authentication that supports OAuth 2.0, not just Keycloak, but I want to bring this as a followup PRs, after we introduce intitial Tornjak components.